qdrant-landing/content/documentation/tutorials/chat-pdf-documents.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Chat With Product PDF Manuals Using Hybrid Search 
+weight: 27
+---
+
+# Chat With Product PDF Manuals Using Hybrid Search 
+
+| Time: 120 min | Level: Advanced | Output: [GitHub](https://github.com/infoslack/qdrant-example/blob/main/HC-demo/HC-DO-LlamaIndex-Jina-v2.ipynb) | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://githubtocolab.com/infoslack/qdrant-example/blob/main/HC-demo/HC-DO-LlamaIndex-Jina-v2.ipynb)   |
+| --- | ----------- | ----------- |----------- |
+
+With the proliferation of digital manuals and the increasing demand for quick and accurate customer support, having a chatbot capable of efficiently parsing through complex PDF documents and delivering precise information can be a game-changer for any business.
+
+In this tutorial, we'll walk you through the process of building a RAG-based chatbot, designed specifically to assist users with understanding the operation of various household appliances.
+We'll cover the essential steps required to build your system, including data ingestion, natural language understanding, and response generation for customer support use cases. 
+
+## Components
+
+- **Embeddings:** Jina Embeddings, served via the [Jina Embeddings API](https://jina.ai/embeddings/#apiform)
+- **Database:** Qdrant, which you can first try in [local mode](https://qdrant.tech/documentation/quick-start/) and the deploy via [Qdrant Hybrid Cloud]()
+- **LLM:** [Mixtral-8x7B-Instruct-v0.1](https://huggingface.co/mistralai/Mixtral-8x7B-Instruct-v0.1) language model on HuggingFace
+- **Framework:** [LlamaIndex](https://www.llamaindex.ai/) for extended RAG functionality and [Hybrid Search support](https://docs.llamaindex.ai/en/stable/examples/vector_stores/qdrant_hybrid/).
```

Comment:
I'd mention Llama parse, as they've recently invested lots of efforts to build it.